content/docs/concepts/network/index.md

```diff
@@ -0,0 +1,437 @@
+configurations for your Istio service mesh. To create a solution for your
+specific mesh, you can combine the configuration objects in a way that best
+fulfills your needs.
+
+You can define the Istio network configuration objects as Kubernetes Custom
+Resource Definition [(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+objects and store their configuration in YAML files. The following sections
+describe the purposes and specific configurations of each configuration object
+and provide examples.
+
+## Virtual Services
+
+You can use virtual services for fine-grained traffic control. Istio can use
+virtual services to configure  ingress, canary, traffic rewrite, or traffic
+policing. A virtual service is not required for simple connectivity. A service
+can map one or more user-addressable destinations to the actual destination
+workloads inside the mesh without a virtual service. The Istio virtual services
+can configure traffic routes to:
+
+-  Other virtual services in the mesh.
+-  Specific services or services' subsets in the mesh.
+-  Other network configuration objects in the mesh.
+
+{{< image width="80%"
+    link="./net-config-2.svg"
+    caption="Possible configurations between virtual services and workloads"
+    >}}
+
+As you can see, virtual services are very flexible and can accommodate a wide
+variety of topologies. With destination rules, virtual services configure the
+traffic routes to your application services. The Istio virtual services can use
+any DNS names with optional wildcard prefixes or CIDR prefixes to create a
+single rule for all matching services. You can address one or more application
+services through a single Istio virtual service. To eliminate redundant rules,
+you can add multiple match conditions to a virtual service configuration. If
+your mesh uses Kubernetes, for example, you can configure a virtual service to
+handle all services in a specific namespace. With the Istio virtual services,
+you can configure routes for the canary and A/B testing for your application's
+services. You can configure each application service version as a virtual
+service subset. You can also configure the corresponding Istio destination rule
+to determine the set of pods or VMs belonging to these subsets.
+
+You can use the Istio virtual services in a variety of ways: to route traffic
+following the rules to provide load balancing to the ingress and egress traffic
+of your mesh with Istio gateways, and to address multiple application services
+through a single Istio virtual service.
+
+The following example shows an Istio virtual service configuration file with
+descriptive value entries. You can use this example as a basis and modify it to
+fit your needs:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: my-vtl-svc
+spec:
+  hosts:
+    - *.my-co.org
+      route:
+      - destination:
+        host: service
```

Comment:
This doesn't seem like a very realistic ... You're delegating everything all my-co.org subdomains to a single monolith service

Reply:
Realism was not my intent. This doc is about adding clarity to how the configuration objects behave. Starting with a simple example where everything goes to a single service is meant as an introduction. Further down the doc, more complex configurations are discussed.